get-events.py

```diff
@@ -76,9 +76,9 @@ def checkFields(self, command, fields):
                 return False
         return True
 
-    def create(self, project_id, summary, description, user_name, label, issue_type='Task'):
+    def create(self, project_id, project_key, summary, description, user_name, label, issue_type='Task'):
```

Comment:
`project_id` isnt being used any more so lets just remove it from here and where it's passed in @everops-davis